tests/std/tests/P0784R7_library_support_for_more_constexpr_containers/test.cpp

```diff
@@ -292,7 +286,7 @@ struct nontrivial_A {
 };
 
 constexpr void test_compiletime_destroy_variants() {
-#ifndef __EDG__ // TRANSITION, VSO-1270011
+#ifndef __EDG__ // TRANSITION, VSO-1327238

```

Comment:
`<sarcasm>`Nothing more satisfying than updating the bug number on a workaround; it gives you such a feeling of progress.`</sarcasm>` (No change requested.)